connectors/airwatch/src/main/service/rhel/files/systemd/airwatch-connector.service

```diff
@@ -5,7 +5,7 @@ After=network.target
 [Service]
 Type=simple
 User=roswell
-ExecStart=/usr/bin/java -jar /opt/vmware/connectors/airwatch/airwatch-connector.jar  --spring.config.additional-location=file:/opt/vmware/connectors/airwatch/application.properties,file:/etc/opt/vmware/connectors/airwatch/application.properties,file:/etc/opt/vmware/connectors/airwatch/managed-apps.yml
+ExecStart=/bin/bash /opt/vmware/connectors/airwatch/start.sh
```

Comment:
I don't think systemd allowed me to just run the script (I had to specify /bin/bash), however, I'll try this again now that I have things working (something else might have been happening yesterday that confused me).

Reply:
And I see some notes that I forgot to follow up on.

So the problem is that my file permissions are wrong on the script.  In the pom, you'll see 660 is applied to the files in this directory.  When I manually chmod to 770, it works w/o the /bin/bash.

Should I bother to fix this properly? (find a better way to handle the file attributes, or put a chmod in the post-install or something)